### PR TITLE
Convert docstrings with inline latex to Markdown

### PR DESCRIPTION
**Pull request by mbauman**

### CODE REVIEW

File: doc/stdlib/math.rst [hunk 1]
```diff
@@ -587,13 +587,13 @@ Mathematical Functions
...
 
-   Compute :math:`\sin(\pi x)` more accurately than ``sin(pi*x)``, especially for large ``x``.
+   Compute :math:`\sin(\pi x)` more accurately than ``sin(pi*x)``\ , especially for large ``x``\ .
 
 .. function:: cospi(x)
 
    .. Docstring generated from Julia source
 
-   Compute :math:`\cos(\pi x)` more accurately than ``cos(pi*x)``, especially for large ``x``.
+   Compute :math:`\cos(\pi x)` more accurately than ``cos(pi*x), especially for large `x``\ .
```
Comment (yuyichao) [inline doc/stdlib/math.rst]: Hmmm. Something is messed up here?

Comment (mbauman) [inline doc/stdlib/math.rst]: Great catch, thank you.  Overzealous deletion of `` ` ``s.

**Comment by yuyichao** [comment]
LGTM other than the one place with missing `` ` ``.

**Comment by mbauman** [comment]
Any ideas why this branch would cause this doctest failure?

```txt
Document: manual/strings
------------------------
**********************************************************************
File "manual/strings.rst", line 291, in default
Failed example:
    s[2]
Expected:
    ERROR: UnicodeError: invalid character index
     in next at ./unicode/utf8.jl:65
     in getindex at strings/basic.jl:37
Got:
    ERROR: UnicodeError: invalid character index
     in next at ./unicode/utf8.jl:65
     in getindex at ./strings/basic.jl:37
**********************************************************************
File "manual/strings.rst", line 296, in default
Failed example:
    s[3]
Expected:
    ERROR: UnicodeError: invalid character index
     in next at ./unicode/utf8.jl:65
     in getindex at strings/basic.jl:37
Got:
    ERROR: UnicodeError: invalid character index
     in next at ./unicode/utf8.jl:65
     in getindex at ./strings/basic.jl:37
**********************************************************************
1 items had failures:
   2 of  82 in default
82 tests in 1 items.
80 passed and 2 failed.
***Test Failed*** 2 failures.
```

**Comment by mbauman** [comment]
Very strange... Here's the minimal patch that changes that backtrace: https://github.com/JuliaLang/julia/compare/JuliaLang:master...mbauman:mb/btbugwtf

**Comment by mbauman** [comment]
Alright, well, I've worked around it.  But, um, WTF.

**Comment by jakebolewski** [comment]
LGTM.

**Comment by tkelman** [comment]
backported in #13072

**Pull request merged**